tony-core/src/main/resources/version-info.properties

```diff
@@ -0,0 +1,9 @@
+#Mon, 03 Dec 2018 16:42:20 -0800
```

Comment:
This file shouldn't be checked in but generated each time as part of the build.

Can you add this file to the .gitignore?

Reply:
You can remove as part of the next PR. I'm merging this one.